src/apps/explorer/components/TransanctionBatchGraph/styled.ts

```diff
@@ -0,0 +1,79 @@
+import { Stylesheet } from 'cytoscape'
+
+import TraderOtherIcon from 'assets/img/TraderOther.svg'
+import CowProtocolIcon from 'assets/img/CoW.svg'
+import DexIcon from 'assets/img/Dex.svg'
+
+export const STYLESHEET: Stylesheet[] = [
+  {
+    selector: 'node[label]',
+    style: {
+      label: 'data(label)',
+      color: '#cfcfcf',
+      height: 50,
+      width: 50,
+      'background-color': '#22232d',
```

Comment:
We could move those colors to the variable files in a section related to the `BatchGraph`